packages/plugin-phone/test/integration/spec/call.js

```diff
@@ -278,23 +527,71 @@ describe(`plugin-phone`, function() {
     });
 
     describe(`#answer()`, () => {
-      it(`accepts an incoming call`);
+      it(`accepts an incoming call`, () => {
+        const call = mccoy.spark.phone.dial(spock.email);
+        return Promise.all([
+          spock.spark.phone.when(`call:incoming`)
+            .then(([c]) => c.answer()),
+          call.when(`connected`)
+            .then(() => assert.equal(call.status, `connected`))
+        ]);
```

Comment:
yes, but not explicitly; there are a lot of redundant code paths in order to keep assertions narrow. that said, with some of the bugs i'm aware of, a lot of these redundancies will be getting refactored